# generated by patch-package 6.5.0 on 2022-03-31 20:04:40
#
# command:
#   npx patch-package @emotion/styled --exclude nothing
#
# declared package:
#   @emotion/styled: 11.8.1
#
diff --git a/node_modules/@emotion/styled/package.json b/node_modules/@emotion/styled/package.json
index fec82fb..a18ff0e 100644
--- a/node_modules/@emotion/styled/package.json
+++ b/node_modules/@emotion/styled/package.json
@@ -49,10 +49,6 @@
     "macro.js.flow"
   ],
   "umd:main": "dist/emotion-styled.umd.min.js",
-  "browser": {
-    "./dist/emotion-styled.cjs.js": "./dist/emotion-styled.browser.cjs.js",
-    "./dist/emotion-styled.esm.js": "./dist/emotion-styled.browser.esm.js"
-  },
   "preconstruct": {
     "umdName": "emotionStyled",
     "entrypoints": [
